modules/core/02-client/client/cli/tx.go

```diff
@@ -455,40 +455,40 @@ func newScheduleIBCUpgradeProposalCmd() *cobra.Command {
 }
 
 // newProvideCounterpartyCmd defines the command to provide the counterparty to an IBC client.
-func newProvideCounterpartyCmd() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "provide-counterparty [client-identifier] [counterparty-client-identifier] [counterparty-merkle-path-prefix]",
-		Args:  cobra.ExactArgs(3),
-		Short: "provide the counterparty to an IBC client",
-		Long: `Provide the counterparty to an IBC client specified by its client ID.
-The [counterparty-merkle-path-prefix] is a comma-separated list of hex-encoded strings.`,
-		Example: fmt.Sprintf("%s tx %s %s provide-counterparty 07-tendermint-0 07-tendermint-1 696263,657572656b61", version.AppName, exported.ModuleName, types.SubModuleName),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			clientIdentifier := args[0]
-			counterpartyClientIdentifier := args[1]
-			counterpartyMerklePathPrefix, err := parseMerklePathPrefix(args[2])
-			if err != nil {
-				return err
-			}
-
-			counterparty := types.NewCounterparty(counterpartyClientIdentifier, counterpartyMerklePathPrefix)
-			msg := types.MsgProvideCounterparty{
-				ClientId:     clientIdentifier,
-				Counterparty: counterparty,
-				Signer:       clientCtx.GetFromAddress().String(),
-			}
-			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
-		},
-	}
-
-	flags.AddTxFlagsToCmd(cmd)
-	return cmd
-}
+// func newProvideCounterpartyCmd() *cobra.Command {
```

Comment:
I commented out this for now because in this PR `MsgProvideCounterparty` has changed and needs more information from the CLI to fill it all in.